### PR TITLE
Westlad/update zokrates

### LINKED ISSUE #141: Update ZoKrates
# Description
The ZoKrates container is approximately six months out of date.  Updating will give access to new crypto library routines which will be useful for future developments.

### PR DESCRIPTION
**Pull request by Westlad**
# Description

This PR updates Nightfall to the latest current ZoKrates container 0.4.11. 

You need to re-run setupAll to make this work!

## Related Issue

fixes #141

## Motivation and Context

The version of ZoKrates that Nightfall uses dates from Jan 2019.  It lacks many new crypto library functions that will be needed for future development of Nightfall, one example being Pedersen hashes.  The current version of ZoKrates has a number of breaking changes, which required the Nightfall code to be updated.

## How Has This Been Tested

It has been tested using the `./nightfall-test` script.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ChaitanyaKonda** [comment]
Tests require a new setup as .code files changed. npm run setupAll is needed before running tests

File: README.md [hunk 1]
```diff
@@ -170,18 +172,18 @@ directories.
...
 ```
--  Mac 
-  - Test suites will open a terminal, where you can see test container's log. And, this terminal will get close
-    automatically.
-  - configure `Teminal.app` to close window when shell exists `exit`.
+
+- Mac
+- Test suites will open a terminal, where you can see test container's log. And, this terminal will
+  get close automatically.
+- configure `Teminal.app` to close window when shell exists `exit`.
```
Comment (iAmMichaelConnor) [inline README.md]: These bullets under the 'Mac' heading aren't indented anymore.

**Comment by ChaitanyaKonda** [comment]
Tested. Code works. Users need to be informed that setup is required to be run once again with this latest code. Between README.md or security-updates.md, not sure which is suitable?

**Comment by iAmMichaelConnor** [comment]
All tests pass. This is fantastic! Just some readme updates needed.

File: README.md (hunk 1 above)
Comment (Westlad) [reply README.md]: fixed

**Pull request merged**